Gordon360/Models/ViewModels/RecIM/ActivityExtendedViewModel.cs

```diff
@@ -29,26 +31,28 @@ public class ActivityExtendedViewModel
 
         public static implicit operator ActivityExtendedViewModel(Activity a)
         {
+            if (a is null) return null;
```

Comment:
Neither the input nor output of this method are marked as nullable, so this is a violation of best practices.

Reply:
This was the issue that I discussed offline with you regarding TeamExtendedViewModel somehow forcing the implicit operator of ActivityExtendedViewModel DESPITE me explicitly calling new TeamExtendedViewModel
![image](https://user-images.githubusercontent.com/78386128/224491289-19b80093-097a-4409-8b86-e292960b3a3c.png)

Currently the match page somehow takes 3-4 seconds to load if there are more than a couple teams in the page. I'm trying to rip out as much unnecessary information as possible so one of the first places was simply truncating the activity from here since it is unnecessary.

Reply:
That makes sense, I'm just saying that you need to add nullability to these types.

Reply:
I just added it in, it's a really REALLY weird bug and I honestly can't even replicate it on other viewmodels so I honestly do not know what is going on.